src/registries/npm-registry.js

```diff
@@ -130,7 +130,7 @@ export default class NpmRegistry extends Registry {
     const possibles = [
       [false, path.join(this.cwd, filename)],
       [true, this.config.userconfig || path.join(userHome, filename)],
-      [false, path.join(getGlobalPrefix(), filename)],
+      [false, path.join(getGlobalPrefix(), 'etc', filename.slice(1))],
```

Comment:
That sound great :) Your change seems okay on Windows too since it seems in-line with what NPM documents and it doesn't break AppVeyor builds.